deploy/complete/helm-chart/mushop/values-mock.yaml

```diff
@@ -2,18 +2,18 @@ global:
   mock:
     service: "all"
 
-# Catalogue service runs in mock mode for product images
-catalogue:
-  enabled: true
+tags:
+  streaming: false
+  atp: false
 
-carts:
```

Comment:
Removing the carts specifics make the carts to the deployed, but because ATP = false will make the carts and orders to automatically use the mock mode?